python/sync_with_meilisearch/README.md

```diff
@@ -0,0 +1,106 @@
+# ⚡ Sync Appwrite to Meilisearch Function
```

Comment:
Can we change this to `⚡ Python Sync with Meilisearch Function`

Reply:
@loks0n I have updated the README